src/helpers/store.js

```diff
@@ -1,10 +1,38 @@
 import Vue from "vue";
 import Vuex from "vuex";
+import * as fb from "./firebaseConfig";
 
 Vue.use(Vuex);
 
 export const store = new Vuex.Store({
-  state: {},
-  actions: {},
-  mutations: {},
+  state: {
+    audio: null,
+    currentUser: null,
+    userProfile: {},
+  },
+  actions: {
+    fetchUserProfile({ commit, state }) {
```

Comment:
Could this function be `async`?

Reply:
I think it could? But would it make much of a difference?

Reply:
Probably not. It seems fine as it stands.